### PR TITLE
hack: copy oc to kubectl

### PR DESCRIPTION
**Pull request by mfojtik**
/cc @smarterclayton

### CODE REVIEW

File: hack/lib/build/binaries.sh [hunk 1]
```diff
@@ -342,14 +354,10 @@ function os::build::place_bins() {
...
       # Create binary copies where specified.
-      local suffix=""
-      if [[ $platform == "windows/amd64" ]]; then
-        suffix=".exe"
-      fi
       for linkname in "${OC_BINARY_COPY[@]}"; do
         local src="${release_binpath}/oc${suffix}"
         if [[ -f "${src}" ]]; then
-          ln "${release_binpath}/oc${suffix}" "${release_binpath}/${linkname}${suffix}"
+          cp "${release_binpath}/oc${suffix}" "${release_binpath}/${linkname}${suffix}"
```
Comment (smarterclayton) [inline hack/lib/build/binaries.sh]: does this need to be cp -f to overwrite the link?

Comment (mfojtik) [reply hack/lib/build/binaries.sh]: sure, fixed

File: hack/lib/build/binaries.sh [hunk 2]
```diff
@@ -330,6 +330,18 @@ function os::build::place_bins() {
         fi
       done
 
+      # Link binaries that we want to link (eg. oc->kubectl)
```
Comment (soltysh) [inline hack/lib/build/binaries.sh]: Why we care about the link, when we'll copy it over a few lines below?

Comment (soltysh) [reply hack/lib/build/binaries.sh]: Nvmd, I haven't seen the dest being different for both.

**Comment by mfojtik** [comment]
@smarterclayton any idea why the gcp job failed?

**Comment by mfojtik** [comment]
@smarterclayton ok, check the last commit... it seems like in bash, the array declaration can't be on the same line as the function invocation, which is ridiculous...

anyway, with the last commit, I think I got kubectl in place where we want it https://gist.github.com/mfojtik/815b18a3859d4f6949fc902d7960e410

**Comment by eparis** [comment]
merged by hand, don't care. I'm a honey badger.

**Pull request merged**